libtcmu.c

```diff
@@ -277,78 +268,43 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	case MODE_SELECT:
 	case MODE_SELECT_10:
 		return tcmu_emulate_mode_select(cdb, iovec, iov_cnt, sense);
+	default:
+		tcmu_err("unknown command %x\n", cdb[0]);
+		return TCMU_NOT_HANDLED;
+	}
+}
+
+static bool command_is_store_call(uint8_t cmd)
+{
```

Comment:
@mikechristie I went through the patch set again and I guess I misunderstood the intention. Please go ahead and I have no objection to this patch set :)